tests/std/tests/GH_005204_regex_collating_ranges/test.cpp

```diff
@@ -0,0 +1,333 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
This new test needs to be added to `tests/std/tests.lst` so the testing infra picks it up.